CONTRIBUTING.md

```diff
@@ -0,0 +1,247 @@
+Contributing to InfluxDB v2
+========================
+
+Bug reports
+---------------
+Before you file an issue, please search existing issues in case it has already been filed, or perhaps even fixed. If you file an issue, please include the following.
+* Full details of your operating system (or distribution) e.g. 64-bit Ubuntu 14.04.
+* The version of InfluxDB you are running
+* Whether you installed it using a pre-built package, or built it from source.
+* If applicable, a small test case or method to reproduce the issue described.
+
+Remember the golden rule of bug reports: **The easier you make it for us to reproduce the problem, the faster it will get fixed.**
+If you have never written a bug report before, or if you want to brush up on your bug reporting skills, we recommend reading [Simon Tatham's essay "How to Report Bugs Effectively."](http://www.chiark.greenend.org.uk/~sgtatham/bugs.html)
+
+TEST CASES SECTION NEEDS REVISION based on v2 API
+
+Test cases should be in the form of `curl` commands. For example:
+```bash
+# create database
+curl -X POST http://localhost:9999/query --data-urlencode "q=CREATE DATABASE mydb"
```

Comment:
These are still incorrect. The 2.x API is different. You don't send it influxql statements. I would look at the swagger, which should have some guidance.

Reply:
@e-dard I could not find examples in any `swagger` file not the whole project, however I could find on the InfluxDB API Doc Ref two examples, I have used them to update the document.

Reply:
In the interests of getting this merged I'm resolving this comment.  We can make iterative changes going forwards.